advancedfx/import_agr.py

```diff
@@ -65,6 +67,8 @@ def readShapes(self):
 	def readSMD(self, filepath, upAxis, rotMode, newscene = False, smd_type = None, target_layer = 0):
 		if SmdImporterEx.bSkipPhysics and smd_type == vs_utils.PHYS:
 			return 0
+		if splitext(basename(filepath))[0].rstrip("123456789").endswith("_lod"):  # skip lod meshes

```

Comment:
This will always be executed the way you wrote it.

You could combine it with the previous if as follows:

```python
if SmdImporterEx.bSkipPhysics and (smd_type == vs_utils.PHYS or splitext(basename(filepath))[0].rstrip("123456789").endswith("_lod")):
```

Also consider renaming `bSkipPhysics` to `bSkipPhysicsAndLod` in order to reflect the function more accurately (or `bSkip`).